Resources/Prototypes/Entities/Objects/Specific/Medical/disease.yml

```diff
@@ -14,6 +14,7 @@
     tags:
     - Recyclable
   - type: DiseaseSwab
+  - type: BotanySwab
```

Comment:
having these on the same entity isn't too bad of an idea but you should change name and descrption

Reply:
I'll update this.